Exercises-4-OOP/src/ExercisesOnClasses/RetangleOneTest.java

```diff
@@ -0,0 +1,16 @@
+package ExercisesOnClasses;
+
+import java.util.Scanner;
+
+public class RetangleOneTest {
```

Comment:
Let's for now not use other solutions. 
First, you need to make the original solution work.

You should look at `TestMain` class for `Rectangle` as the client's requirements, meaning the `TestMain` should pass!
You can implement the `Rectangle` however you want, but the clients expect it to behave as in  `TestMain`.
Don't create your own test classes, use the one given to you!

Reply:
Please remove `RectangleOne` and `RectangleOneTest` classes as they are not related to the exercise.

Reply:
Done.